pkg/cmd/roachtest/tests/network.go

```diff
@@ -276,6 +279,15 @@ sudo iptables-save
 		// (attempt to) restore iptables when test end, so that cluster
 		// can be investigated afterwards.
 		defer func() {
+			// Check that iptable DROP actually blocked traffic.
+			t.L().Printf("verify that traffic to node one is blocked")
+			packetsDropped, err := iptablesPacketsDropped(ctx, t.L(), c, c.Node(expectedLeaseholder))
+			if err != nil {
+				t.Fatal(err)
+			}
+			if packetsDropped == 0 {
+				t.Fatal(errors.Errorf("Expected node 1 to be partitioned but reported no packets dropped."))
```

Comment:
Same point about `expectedLeaseholder`. Also, no need to wrap the argument to `Fatal` into an error, since `Fatal` already does that. You can just call `t.Fatalf("format", args...)`.